### ISSUE TITLE
Adapt distribute_flowline tutorial

### ISSUE THREAD

**Issue by pat-schmitt** [OGGM/tutorials#118]
- Add an explanation of new merging functionality to this tutorial, added in PR https://github.com/OGGM/oggm/pull/1674.
- Also need to rename `distributed_thickness` to `simulated_thickness` in this notebook.

**Comment by bearecinos** [comment]
Hey @pat-schmitt I found another issue with the tutorials ... in [distribute_thickness_from_simulation](https://github.com/pat-schmitt/oggm/blob/744ccfe502e8860e600294b9852d0d6754a082ae/oggm/sandbox/distribute_2d.py#L228C1-L231C1) you assume that ``add_smoothed_glacier_topo`` and ``assign_points_to_band`` have been run before that task but the tutorials do not reflect that. 
```
 # This is to add a new topography to the file (smoothed differently)
workflow.execute_entity_task(distribute_2d.add_smoothed_glacier_topo, gdirs)
# This is to get the bed map at the start of the simulation
workflow.execute_entity_task(tasks.distribute_thickness_per_altitude, gdirs)
# This is to prepare the glacier directory for the interpolation (needs to be done only once)
workflow.execute_entity_task(distribute_2d.assign_points_to_band, gdirs);
```

Maybe modify the order above in the notebook. Took me a bit to work it out.
